gists/pull-request-guidelines.md

```diff
@@ -16,6 +16,7 @@ Guidelines for creating pull requests at LaunchPad Lab for three types of users:
 - Provide screenshots and instructions to reproduce / find style issues if there are outstanding items that require design assistance
 - Add guidance about what the reviewer(s) should focus on
 - Document what the QA tester should be testing
+- Review the diff and ensure that all changes are relevant and intentional
```

Comment:
one other thing that comes to mind, "relevant and intentional, and address requirements of listed issue or ticket" or something along those lines

Reply:
this is getting at both - did you cover off on the requirements of the ticket? (other than what's out of scope) and / or are there items in this diff that don't pertain to the requirements the PR or branch was meant to address